### PR TITLE
Added ability to automatically choose contrasting color for titleBar.…

### PR DESCRIPTION
**Pull request by fyrebase**
This PR changes the title text color so it contrasts with the title background color

### CODE REVIEW

**Comment by oreporan** [comment]
I like it :)

Can you do the same for `statusBar.foreground` ?

Thanks

File: src/changeColors.ts [hunk 1]
```diff
@@ -3,6 +3,14 @@ import getSettings from './getSettings';
 
 type ColorCustomization = { [key: string]: string | undefined };
 
+const invertHex = (hex) => {
```
Comment (oreporan) [inline src/changeColors.ts]: `const invertHex = hex => hex && hex.replace('#', '0x') > 0xffffff / 2 ? '#000000' : '#ffffff';`

**Review by oreporan — APPROVED** [review]

**Comment by fyrebase** [comment]
Added automatic contrast to activity bar and status bar as well ;)

File: src/changeColors.ts [hunk 2]
```diff
@@ -15,16 +23,22 @@ export default async (color?: string) => {
 
     const tabBarBorderColor = (tabBorder !== false) ? color : undefined;
     const titleBarBackgroundColor = titleBackground ? color : undefined;
+    const titleBarForegroundColor = titleBackground ? invertHex(color) : undefined;
```
Comment (oreporan) [inline src/changeColors.ts]: just for sake of redundant function calls, can you set invertHex at the top

**Review by oreporan — CHANGES_REQUESTED** [review]

**Pull request merged**